### PR TITLE
Added a note about session data size in PdoSessionHandler

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | #5269 |

### CODE REVIEW

File: cookbook/configuration/pdo_session_storage.rst [hunk 1]
```diff
@@ -172,15 +172,18 @@ of your project's data, you can use the connection settings from the
             '%database_password%',
         ));
 
-Example SQL Statements
-----------------------
+.. example-sql-statements::
```
Comment (xabbuh) [inline cookbook/configuration/pdo_session_storage.rst]: should be:

```rst
.. _example-sql-statements:
```

File: cookbook/configuration/pdo_session_storage.rst [hunk 2]
```diff
@@ -225,3 +224,16 @@ For MSSQL, the statement might look like the following:
...
+    If the session data doesn't fit in the data column, it might get truncated
+    by the database engine. To make matters worse, when the session data gets
+    corrupted, PHP ignores the data without giving a warning.
+
+    If the application stores large amounts of session data, this problem can
+    be solved by increasing the column size (use ``BLOB`` or even ``MEDIUMBLOB``).
+    When using MySQL as the database engine, you can also enable the `strict SQL mode`_
+    to get noticed when such an error happens.
+
+.. _`strict SQL mode`: https://dev.mysql.com/doc/refman/5.0/en/sql-mode.html
```
Comment (xabbuh) [inline cookbook/configuration/pdo_session_storage.rst]: we should link to a more up-to-date version

Comment (javiereguiluz) [inline cookbook/configuration/pdo_session_storage.rst]: Linked to 5.7 version. Thanks.

File: cookbook/configuration/pdo_session_storage.rst (hunk 1 above)
Comment (javiereguiluz) [inline cookbook/configuration/pdo_session_storage.rst]: Fixed. Thanks.

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thanks @javiereguiluz! I've changed the directive to a caution directive in fc9dd41, as I think it better fits the content (and it is actually quite important information). As always, let me know if you don't agree with that change :)

**Pull request closed without merging**